test/Analysis/Go/Spec.hs

```diff
@@ -6,8 +7,8 @@ import qualified Data.Language as Language
 import SpecHelpers
 
 
-spec :: TaskSession -> Spec
-spec session = parallel $ do
+spec :: (?session :: TaskSession) => Spec
```

Comment:
You could define this as a constraint synonym somewhere and lose the `LANGUAGE` pragma as well as simplifying the syntax, rather like `HasCallStack`. (You’d also need a `currentSession` binding which uses it.)